### PR TITLE
[5.1] Check if model update was actually performed

### PR DESCRIPTION
**Pull request by FranPregernik**
laravel/framework#10704

### CODE REVIEW

File: src/Illuminate/Database/Connection.php [hunk 1]
```diff
@@ -352,7 +352,7 @@ public function insert($query, $bindings = [])
      *
      * @param  string  $query
      * @param  array   $bindings
-     * @return int
+     * @return int Number of rows affected by the query
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: don't add that comment please

File: src/Illuminate/Database/Connection.php [hunk 2]
```diff
@@ -364,7 +364,7 @@ public function update($query, $bindings = [])
...
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connection.php]: don't add that comment please

File: src/Illuminate/Database/ConnectionInterface.php [hunk 3]
```diff
@@ -54,7 +54,7 @@ public function insert($query, $bindings = []);
      *
      * @param  string  $query
      * @param  array   $bindings
-     * @return int
+     * @return int Number of rows affected by the query
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/ConnectionInterface.php]: etc

**Comment by GrahamCampbell** [comment]
Looks like a BC break to me. Probably should go to 5.2.

**Comment by FranPregernik** [comment]
So master branch?

**Comment by GrahamCampbell** [comment]
Yes. Ping @taylorotwell.

**Comment by dan-har** [comment]
:thumbsup: very useful.

But don't forget the [```save()``` method](https://github.com/FranPregernik/framework/blob/edd357dafd804b61446537e7c5f22ddac7bef271/src/Illuminate/Database/Eloquent/Model.php#L1491), the return type need to be changed.
Also, maybe worth adding to docs that if ```touchOwners``` is used, it wont count in the number of updated rows.

**Comment by FranPregernik** [comment]
@dan-har I might be missing something but on branch 5.1 the [save()](https://github.com/FranPregernik/framework/blob/edd357dafd804b61446537e7c5f22ddac7bef271/src/Illuminate/Database/Eloquent/Model.php#L1491) method return type is bool and [performUpdate()](https://github.com/FranPregernik/framework/blob/edd357dafd804b61446537e7c5f22ddac7bef271/src/Illuminate/Database/Eloquent/Model.php#L1547) did not change the return type. Why do I need to change the return type?

**Comment by dan-har** [comment]
I was missing something...,  my comment before is not relevant.

**Pull request merged**